cluster/docker-cluster/kube-config/master-multi.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Sorry, old comments are missing. I want to make master configurable. So if user set `MASTER_CONF=yes`, and customized this file, we will mount this file in and replace the original master pod config.